src/receipt/_style.scss

```diff
@@ -0,0 +1,67 @@
+#receipt-container {
+    width: 1170px;
+    padding-top: 2rem;
+}
+
+.page__receipt {
+    h2 {
+        margin-bottom: 10px;
+        font-weight: 400;
+    }
+
+    .thank-you {
+        font-size: 38px;
+    }
+
+    .info-item {
+        overflow: hidden;
+    }
+
+    .copy {
+        > div {
+            margin-bottom: 1rem;
+            line-height: 1.5rem;
+        }
+
+        .billing-address {
+            margin: 2rem 0;
+        }
+    }
+
+    .credit-messaging {
+        margin-top: 2rem;
+    }
+
+    .enterprise-message {
+        margin-bottom: 2rem;
+    }
+
+    .order-total {
+        font-size: 0.875rem;
```

Comment:
This is the `small` type ramp in our design system. This could be replaced with `$font-size-sm` or use a `small` utility class name on the element itself.